gravity-forms/gw-gravity-forms-custom-js.php

```diff
@@ -91,12 +91,18 @@ public function noconflict_styles( $scripts = array() ) {
 
 	public function add_custom_js_setting( $settings, $form ) {
 
+		// GF 2.5 may fire `gform_form_settings` before `save_custom_js_setting`
+		$customJS = rgar( $form, 'customJS' );
+		$postJS   = esc_html( rgpost( 'custom_js' ) );
```

Comment:
These variables should be `$custom_js` and `$post_js` respectively (see our [PHP naming conventions](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#naming-conventions)).

Reply:
Ah yes, great catch! It was a JS heavy day 😅